template-component/package.json

```diff
@@ -1,12 +1,6 @@
 {
-  "name": "@convex-dev/sharded-counter",
```

Comment:
The idea here was to show how we'd like these to be named and how to link them up, there's even a script to rename these strings https://github.com/get-convex/templates/blob/main/template-component/rename.mjs

<img width="1258" height="915" alt="Image" src="https://github.com/user-attachments/assets/ba291178-ff04-40bb-8e19-f9b00e8e89e9" />

Reply:
Sure, but we don't officially support other people making components yet; we're probably still the primary users of this until the component authoring API is stable? @ianmacartney and @jordanhunt22 is thinking about timelines here but I'd just leave it until this is ready